draft-vvv-webtransport-quic.md

```diff
@@ -237,21 +241,20 @@ The user agent MUST send the "Origin" field.  The "Origin" field MUST be set to
 the origin of the client initiating the connection, serialized as described in
 the "serializing a request origin" section of [FETCH].
 
+The server indicates that it accepts the origin by sending its header block and
+a FIN on the stream 0.  If the server does not accept the origin, it MUST close
+the connection.
```

Comment:
With the current format and defined fields, this means just sending an empty STREAM frame with FIN?

Reply:
Correct.